testutil/integration/app.go

```diff
@@ -591,6 +594,19 @@ func NewCompleteIntegrationApp(t *testing.T) *App {
 	err = bankKeeper.MintCoins(integrationApp.sdkCtx, apptypes.ModuleName, moduleBaseMint)
 	require.NoError(t, err)
 
+	// Fund the supplier operator account to be able to submit proofs
+	err = bankKeeper.SendCoinsFromModuleToAccount(
```

Comment:
Can you create a helper called "fundAccount" that mints new tokens in the banker module and sends it over?